packages/core/src/pubsub/pubsub-message.ts

```diff
@@ -124,6 +130,11 @@ export function deserialize(message: any): PubsubMessage {
         typ: MsgType.UPDATE,
         stream,
         tip: toCID(parsed.tip),
+        /*
+          @active-branch
+          placeholder, optional type
+        */
+        model: parsed.model
```

Comment:
you'll need to parse the `model` into a StreamID like is done for the `stream` field above